### PR TITLE
Feature/dataapi model

### PR DESCRIPTION
**Pull request by tributech-dhofer**

### CODE REVIEW

File: DTDL/options_persistence.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          }
+        ]
+      },
+      "writable": true,
+      "displayName": "Data API persistence provider",
+      "description": "Select a persistence provider for the stream."
+    },
+    {
+      "@type": "Property",
+      "name": "PersistType",
```
Comment (simonpfeifhofer) [inline DTDL/options_persistence.json:33]: @tributech-dhofer @mayrmax should we align it with `encoding` in `base_stream.json`?

File: DTDL/options_publish.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      "@type": "Property",
+      "name": "PublishToMessageBus",
+      "schema": "boolean",
+      "writable": true,
+      "displayName": "Publish to message bus",
+      "description": "Defines wether or not the stream is going to be published on the message bus."
+    },
+    {
+      "@type": "Property",
+      "name": "PublishType",
```
Comment (simonpfeifhofer) [inline DTDL/options_publish.json:18]: @tributech-dhofer @mayrmax same question.

File: DTDL/options_persistence.json (hunk 1 above)
Comment (simonpfeifhofer) [reply DTDL/options_persistence.json:33]: As discussed via call with @mayrmax its ok for now.

File: DTDL/options_publish.json (hunk 2 above)
Comment (simonpfeifhofer) [reply DTDL/options_publish.json:18]: As discussed via call with @mayrmax it's ok for now.

**Review by simonpfeifhofer — APPROVED** [review]

**Review by mayrmax — APPROVED** [review]

**Pull request merged**